Xamarin.PropertyEditing/ViewModels/BrushPropertyViewModel.cs

```diff
@@ -47,7 +48,11 @@ public BrushPropertyViewModel (TargetPlatform platform, IPropertyInfo property,
 		protected override async Task UpdateCurrentValueAsync ()
```

Comment:
This isn't the right override for these property change notifications. A request to update the current value does not necessarily mean the value changes. `OnValueChanged()` is the proper override.